src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -144,4 +144,17 @@
      * @return The connection
      */
     PlayerConnection getConnection();
+
+    /**
+     * Kicks the player.
```

Comment:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  adam@eq2.serveftp.net
    Domain eq2.co.uk has exceeded the max emails per hour (61/60 (101%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext1.iad.github.net ([192.30.252.192]:42033 helo=github-smtp2a-ext-cp1-prd.iad.github.net)
	by web3.cov.yesit.co.uk with esmtps (TLSv1.2:DHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.84)
	(envelope-from <noreply@github.com>)
	id 1YJ5A4-0006vy-1y
	for adam@eq2.co.uk; Wed, 04 Feb 2015 18:58:52 +0000
Date: Wed, 04 Feb 2015 10:58:52 -0800
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1423076332;
	bh=km2t+TE7/Qdt29B9RU8GOZuInIEJqk9esXRTWVtf1s8=;
	h=From:Reply-To:To:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=UyDAY5rXAqOEFVFjfsdqZXhDAV+XoZbGspQOy84kAjakkm8D4diiPxvq4tmHWqylO
	 V8QtXNUbvtGDMgBzT5Nc8+GvpcjEKWnyOMrHHgQ58Ux8RanP6beI0liw+/MmjSXZf/
	 pSYIEPmJYvu3bSeo7UIyM52IyFJ7jNvtFQU961aA=
From: gratimax <notifications@github.com>
Reply-To: SpongePowered/SpongeAPI <reply@reply.github.com>
To: SpongePowered/SpongeAPI <SpongeAPI@noreply.github.com>
Message-ID: <SpongePowered/SpongeAPI/pull/412/r24108785@github.com>
In-Reply-To: <SpongePowered/SpongeAPI/pull/412@github.com>
References: <SpongePowered/SpongeAPI/pull/412@github.com>
Subject: Re: [SpongeAPI] Bans API (#412)
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_54d26bec457a3_5a6c3fe920d5d2b86922a0";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
Precedence: list
X-GitHub-Sender: gratimax
X-GitHub-Recipient: Mumfrey
List-ID: SpongePowered/SpongeAPI <SpongeAPI.SpongePowered.github.com>
List-Archive: https://github.com/SpongePowered/SpongeAPI
List-Post: <mailto:reply@reply.github.com>
List-Unsubscribe: <mailto:unsub+001180bf1e5b688f518a5605a896c59abff710d45aadbe0b92cf0000000110ea2dec92a170ce016fdef1@reply.github.com>,
 <https://github.com/notifications/unsubscribe/ABGAvxhUa3JEgU2kG_wv-ffDaFxKAJZqks5nomNsgaJpZM4DW2RI>
X-Auto-Response-Suppress: All
X-GitHub-Recipient-Address: adam@eq2.co.uk


----==_mimepart_54d26bec457a3_5a6c3fe920d5d2b86922a0
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

> @@ -144,4 +144,17 @@
>       * @return The connection
>       */
>      PlayerConnection getConnection();
> +
> +    /**
> +     * Kicks the player.

@Aaron1011 what would you suggest as the default behavior, then? `You were kicked`? Or maybe the javadocs should say that the behavior is undefined.

---
Reply to this email directly or view it on GitHub:
https://github.com/SpongePowered/SpongeAPI/pull/412/files#r24108785
----==_mimepart_54d26bec457a3_5a6c3fe920d5d2b86922a0
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>In <a href="https://github.com/SpongePowered/SpongeAPI/pull/412#discussion_r24108785">src/main/java/org/spongepowered/api/entity/player/Player.java</a>:</p>
<pre style='color:#555'>&gt; @@ -144,4 +144,17 @@
&gt;       * @return The connection
&gt;       */
&gt;      PlayerConnection getConnection();
&gt; +
&gt; +    /**
&gt; +     * Kicks the player.
</pre>
<p><a href="https://github.com/Aaron1011" class="user-mention">@Aaron1011</a> what would you suggest as the default behavior, then? <code>You were kicked</code>? Or maybe the javadocs should say that the behavior is undefined.</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>Reply to this email directly or <a href="https://github.com/SpongePowered/SpongeAPI/pull/412/files#r24108785">view it on GitHub</a>.<img alt="" height="1" src="https://github.com/notifications/beacon/ABGAv8iOzRDjYWqCwGnQPdJJ41YHuOb0ks5nomNsgaJpZM4DW2RI.gif" width="1" /></p>
<div itemscope itemtype="http://schema.org/EmailMessage">
  <div itemprop="action" itemscope itemtype="http://schema.org/ViewAction">
    <link itemprop="url" href="https://github.com/SpongePowered/SpongeAPI/pull/412/files#r24108785"></link>
    <meta itemprop="name" content="View Pull Request"></meta>
  </div>
  <meta itemprop="description" content="View this Pull Request on GitHub"></meta>
</div>

----==_mimepart_54d26bec457a3_5a6c3fe920d5d2b86922a0--